point-of-sale-app/frontend/src/main/java/com/google/abmedge/frontend/FrontendApplication.java

```diff
@@ -0,0 +1,22 @@
+package com.google.abmedge.frontend;
```

Comment:
All of the source code requires copywrite headers (or at least used to unless things changed)

Reply:
Added them in https://github.com/GoogleCloudPlatform/anthos-edge-usecases/pull/1/commits/f2084a5fa77ba7391228e0f2ee4f61f0f6db4eea.